### PR TITLE
New emergency shuttle, The Tranquility Relocation Shuttle!

### PR DESCRIPTION
**Pull request by spockye**
## About The Pull Request
The Tranquility Relocation Shuttle:tm: is a decently sized emergency shuttle, containing a bar, a library, a small chapel, a security bay, a medical bay, two "Flora pools",  a large bridge, and an engineering bay. The perfect way to end a peaceful shift! (totally)

The name, comes from, well, it's purpouse, as a pretty tranquil place!

image
### Mapping March
Ckey to receive rewards: Spockye

## Why It's Good For The Game
More shuttle variety is always good! i'd also say this one is pretty unique
## Changelog
:cl:
add: Added a new emergency shuttle, the emergency_tranquility
/:cl:

### CODE REVIEW

File: config/unbuyableshuttles.txt [hunk 1]
```diff
@@ -31,3 +31,4 @@
 #_maps/shuttles/emergency_supermatter.dmm
 #_maps/shuttles/emergency_venture.dmm
 #_maps/shuttles/emergency_wabbajack.dmm
+#_maps/shuttles/emergency_tranquility.dmm
```
Comment (Fikou) [inline config/unbuyableshuttles.txt:34]: abc order

Comment (spockye) [reply config/unbuyableshuttles.txt:34]: sorry, i don't know my abc's :(

**Comment by Helg2** [comment]
Tranquility shuttle with hygiene bots, kek

**Comment by spockye** [comment]
> Tranquility shuttle with hygiene bots, kek

im afraid unwashed, bloodcovered, stinky spessmen is harmful to the shuttles tranquility. please do not resist

File: config/unbuyableshuttles.txt [hunk 2]
```diff
@@ -29,5 +29,7 @@
 #_maps/shuttles/emergency_russiafightpit.dmm
 #_maps/shuttles/emergency_scrapheap.dmm
 #_maps/shuttles/emergency_supermatter.dmm
+#_maps/shuttles/emergency_tranquility.dmm
 #_maps/shuttles/emergency_venture.dmm
 #_maps/shuttles/emergency_wabbajack.dmm
+
```
Comment (Fikou) [inline config/unbuyableshuttles.txt:35]:
```suggestion
```

**Review by Maurukas — APPROVED** [review]

**Pull request merged**